packages/mobile/locales/es-419/nuxVerification2.json

```diff
@@ -1,30 +1,30 @@
 {
-  "getVerified": "Verificar el Teléfono",
+  "getVerified": "Confirmar el Teléfono",
   "notReadyForCode": "Aún no está listo para ingresar el código",
   "emptyVerificationCode": "Sin Código en el Portapapeles",
-  "invalidVerificationCode": "Código de Verification Inválido",
-  "repeatVerificationCode": "Código de Verificación Repetido",
+  "invalidVerificationCode": "Código de Confirmación Inválido",
+  "repeatVerificationCode": "Código de confirmación Repetido",
   "revealAttestationFailure": "Error al solicitar mensaje SMS",
-  "verificationFailure": "Fallo en la Verificación",
+  "verificationFailure": "Fallo en la confirmación",
   "cantVerifyRevokedAccount": "Esta cuenta ha sido revocada previamente. Crea una nueva cuenta e inténtalo de nuevo.",
-  "verificationTimeout": "Verificación no pudo ser verificada a tiempo",
+  "verificationTimeout": "Confirmación no pudo ser confirmado a tiempo",
   "errorRequestCode": "Error al solicitar los códigos.",
   "errorRedeemingCode": "Error al canjear el código.",
-  "pleaseRetry": "Por favor reinicie la verificación.",
-  "retryVerification": "Reintentar la verificación",
+  "pleaseRetry": "Por favor reinicie la confirmación.",
+  "retryVerification": "Reintentar la confirmación",
   "education": {
-    "header": "Verificar el celular",
-    "body1": "Te solicitamos que verifiques tu número telefónico.",
+    "header": "Confirmar el celular",
+    "body1": "Te solicitamos que confirmar tu número telefónico.",
     "body2": "Esto facilita el envío y la recepción de valores entre amigos. Puede omitir este paso ahora y hacerlo después.",
-    "learnMore": "Más información sobre la verificación telefónica",
-    "start": "Comenzar la verificación",
+    "learnMore": "Más información sobre la confirmación telefónica",
```

Comment:
I think it would read better if it said `Más información sobre la confirmación del número telefónico`